frontend/src/features/navigation/header/recipesMenu.tsx

```diff
@@ -0,0 +1,95 @@
+import { KeyboardArrowDownOutlined, KeyboardArrowUpOutlined } from '@mui/icons-material';
+
+import Button from '@mui/material/Button';
+import ClickAwayListener from '@mui/material/ClickAwayListener';
+import Grow from '@mui/material/Grow';
+import { Link } from 'react-router-dom';
+import MenuItem from '@mui/material/MenuItem';
+import MenuList from '@mui/material/MenuList';
+import Paper from '@mui/material/Paper';
+import Popper from '@mui/material/Popper';
+import React from 'react';
+import { styled } from '@mui/material/styles';
+
+const menuItems = [
+  { label: '🍽️ All recipes', path: '/recipes/all' },
+  { label: '🥪 Breakfasts', path: '/recipes/breakfasts' },
+  { label: '🍔 Lunch', path: '/recipes/lunch' },
+  { label: '🥐 Baking', path: '/recipes/baking' },
+  { label: '🧁 Desserts', path: '/recipes/desserts' },
+  { label: '🍹 Drinks', path: '/recipes/drinks' },
+  { label: '🍿 Snacks', path: '/recipes/snacks' },
+  { label: '🥗 Salads', path: '/recipes/salads' },
+];
+
+const StyledButton = styled(Button)(({ theme, clicked }) => ({
+  display: 'flex',
+  alignItems: 'center',
+  padding: theme.spacing(1),
+  textTransform: 'none',
+  backgroundColor: clicked ? theme.palette.action.hover : 'transparent',
+  color: theme.palette.text.secondary,
+  '&:hover': {
+    backgroundColor: theme.palette.action.hover,
+  },
+}));
+
+const Container = styled('div')(() => ({
+  display: 'flex',
+  alignItems: 'center',
+}));
+
+const RecipesMenu = () => {
+  const [open, setOpen] = React.useState(false);
+  const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
+  const [clicked, setClicked] = React.useState(false);
+
+  const handleOpen = (event: React.MouseEvent<HTMLElement>) => {
+    setAnchorEl(event.currentTarget);
+    setOpen(true);
```

Comment:
It seems (correct me if I am wrong pls) that you don't need the "open" and "clicked" states and you can use just one of them. You are setting both of them to the same same values and initialize them with the same value as well - so where is the difference?